### PR TITLE
Syndication UI work

### PR DESCRIPTION
**Pull request by darconny**

### CODE REVIEW

File: client/app/scripts/liveblog-syndication/styles/syndication.less [hunk 1]
```diff
@@ -1,107 +1,118 @@
...
+        color: #444;
         border-bottom: 1px solid #e0e0e0;
         font-weight: 400;
         overflow: hidden;
+        margin-top: 1px;
 
         button {
             float: left;
-            background: transparent;
+            background: transparent url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIyNiIgaGVpZ2h0PSIyNiIgdmlld0JveD0iMCAwIDI2IDI2IiBlbmFibGUtYmFja2dyb3VuZD0ibmV3IDAgMCAyNiAyNiI+PHBhdGggZmlsbD0iIzdjN2M3YyIgZD0ibTUuMjkyIDEyLjI5NGw2LTZjLjM4OS0uMzg5IDEuMDI1LS4zODkgMS40MTQgMCAuMzg5LjM4OS4zODkgMS4wMjUgMCAxLjQxNGwtNC4yOTMgNC4yOTNoMTIuNTg2Yy41NTIgMCAxIC40NDggMSAxIDAgLjU1Mi0uNDQ4IDEtMSAxaC0xMi41ODZsNC4yOTIgNC4yOTJjLjM4OS4zODkuMzg5IDEuMDI1IDAgMS40MTRzLTEuMDI1LjM4OS0xLjQxNCAwYzAgMC01Ljk5OS01Ljk5Ni02LTUuOTk4LS4zODgtLjM4OS0uMzg4LTEuMDI1LjAwMDEtMS40MTQiLz48L3N2Zz4=) no-repeat center center;
```
Comment (m0g) [inline client/app/scripts/liveblog-syndication/styles/syndication.less]: Hey, what's the benefit of doing that?

**Comment by mscam** [comment]
:+1:

File: client/app/scripts/liveblog-syndication/styles/syndication.less (hunk 1 above)
Comment (darconny) [inline client/app/scripts/liveblog-syndication/styles/syndication.less]: Hey, this is just an arrow copied from SD (with data uri svg).

Comment (m0g) [inline client/app/scripts/liveblog-syndication/styles/syndication.less]: ok

**Pull request merged**